packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/delete-dialog.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Dialog, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+type TDeleteDialog = {
+    is_running: boolean;
+    is_delete_modal_open: boolean;
+    onToggleDeleteDialog: (type: string, param: boolean) => void;
+};
+
+const DeleteDialog = ({ is_delete_modal_open, is_running, onToggleDeleteDialog }: TDeleteDialog) => {
+    const onHandleChange = (type: string, param: boolean) => {
+        onToggleDeleteDialog(type, false);
+    };
```

Comment:
```suggestion
    };
    
```
nit: space for readability

Reply:
done ✅